core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPushdownLimitIntoWindow.java

```diff
@@ -172,4 +125,69 @@ public void testMultipleWindowFunctions()
                         Optional.empty()),
                 false);
     }
+
+    private void assertLimitAboveWindow(String rankingFunctionName)
```

Comment:
Just as in the other PR, I recommend putting the assert* methods just below the test* methods that will be using them because they have a direct relationship.